### PR TITLE
Conda recipe for Modin

### LINKED ISSUE #1826: Building and packaging for omnisci and modin
## The goal is user-friendly installation of Modin with all the dependencies:

- [ ]   conda install -c intel modin
- [ ]   pip install modin[all] 

It implies:

- [ ]  Adding conda-forge/ray-feedstock
- [ ]  Adding omniscidb wheels to pypi (optional)
- [x]  Adding -c intel omniscidb packages
- [x]  Adding -c intel modin
- [ ]  Up-streaming omnisci recipes

### PR DESCRIPTION
**Pull request by anton-malakhov**
## Add conda recipe for Intel Modin releases

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #1826 and #1948
- [x] tests added and passing

### CODE REVIEW

File: ci/conda-recipe/meta.yaml
Comment (devin-petersohn) [inline ci/conda-recipe/meta.yaml:1]: Does this file belong in `ci`? I think there might be a better place for it in something like `build` or `packaging`.

Comment (anton-malakhov) [reply ci/conda-recipe/meta.yaml:1]: well, there is `ci/benchmarks` directory already which have the same or even less level of correlation with CI/CD as conda-recipe. I'll put it into `scripts/` - like in omniscidb

**Review by devin-petersohn — COMMENTED** [review]
LGTM, is this still blocked on #1871?

**Comment by anton-malakhov** [comment]
it is logical continuation or even part of #1871 but can be merged right away if you prefer

**Review by devin-petersohn — APPROVED** [review]
Okay to merge now. Thanks @anton-malakhov

File: scripts/conda-recipe/meta.yaml
Comment (heuermh) [inline scripts/conda-recipe/meta.yaml:30]: Sorry for not catching this before, should not`modin.experemental.pandas` be `modin.experimental.pandas`?

Comment (vnlitvinov) [reply scripts/conda-recipe/meta.yaml:30]: I guess it should, cc @anton-malakhov

**Pull request merged**
